### PR TITLE
Align UI changes in flaw create page

### PR DESCRIPTION
**Pull request by RichardXuan**

### CODE REVIEW

File: features/pages/flaw_create_page.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        "reportedDateEditBtn": ("XPATH", "(//button[@class='osim-editable-date-pen input-group-text'])[1]"),
+        "reportedDateInput": ("XPATH", "(//input[@class='form-control is-invalid'])[1]"),
+
+        "publicDateEditBtn": ("XPATH", "(//button[@class='osim-editable-date-pen input-group-text'])[2]"),
+        "publicDateInput": ("XPATH", "(//input[@class='form-control is-invalid'])[1]"),
+
+        "cweidEditBtn": ("XPATH", "(//button[@class='osim-editable-text-pen input-group-text'])[5]"),
+        "cweidInput": ("XPATH", "(//input[@class='form-control'])[6]"),
+    }
```
Comment (huiwangredhat) [inline features/pages/flaw_create_page.py:46]: So as we talked about today,  do the locators also need improvement?

**Review by huiwangredhat — COMMENTED** [review]
The step " Then A new flaw is created" was failed due to ElementNotInteractableException.

Detail: 
_File "features/steps/flaw_create.py", line 73, in step_impl
          check_created_flaw_exist(context)
        File "features/steps/flaw_create.py", line 51, in check_created_flaw_exist
          go_to_advanced_search_page(context.browser)
        File "/Users/huiwang/create_flaw/osim/features/utils.py", line 152, in go_to_advanced_search_page
          home_page.click_btn("advancedSearchDropDownBtn")
        File "/Users/huiwang/create_flaw/osim/features/pages/home_page.py", line 113, in click_btn
          element.click_button()...
selenium.common.exceptions.ElementNotInteractableException: Message: Element <button class="btn btn-secondary dropdown-toggle" type="button"> could not be scrolled into view_

**Comment by huiwangredhat** [comment]
The step "When I create new embargoed flaw with valid data" failed due to  selenium.common.exceptions.ElementClickInterceptedException.
Detail:_File "features/steps/flaw_create.py", line 79, in step_impl
          create_flaw_with_valid_data(context, embargoed=True)
        File "features/steps/flaw_create.py", line 39, in create_flaw_with_valid_data
          flaw_create_page.click_btn('createNewFlawBtn')
        File "/Users/huiwang/create_flaw/osim/features/pages/flaw_detail_page.py", line 234, in click_btn
          element.click_button()...
selenium.common.exceptions.ElementClickInterceptedException: Message: Element <button class="btn btn-primary col" type="submit"> is not clickable at point (1235,837) because another element <main class="osim-page-view">
_

**Comment by huiwangredhat** [comment]
The step "Then A new flaw is created" due to ElementNotInteractableException.
Detail:
_File "features/steps/flaw_create.py", line 73, in step_impl
          check_created_flaw_exist(context)
        File "features/steps/flaw_create.py", line 51, in check_created_flaw_exist
          go_to_advanced_search_page(context.browser)
        File "/Users/huiwang/create_flaw/osim/features/utils.py", line 152, in go_to_advanced_search_page
          home_page.click_btn("advancedSearchDropDownBtn")
        File "/Users/huiwang/create_flaw/osim/features/pages/home_page.py", line 113, in click_btn
          element.click_button()...
selenium.common.exceptions.ElementNotInteractableException: Message: Element <button class="btn btn-secondary dropdown-toggle" type="button"> could not be scrolled into view_

File: features/pages/flaw_create_page.py (hunk 1 above)
Comment (RichardXuan) [reply features/pages/flaw_create_page.py:46]: I think we should do all related improvement in another PR.

**Review by huiwangredhat — APPROVED** [review]
LGTM. The locators will be updated in other PR.

**Pull request merged**